Python: Getting Started logging (#7860)
getsentry/sentry-docs: src/platforms/python/guides/logging/index.mdx

```

## Configure

The logging integrations is a default integration so it will be enabled automatically when you initialize the Sentry SDK.

<SignInNote />

```python
import sentry_sdk

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for performance monitoring.
    traces_sample_rate=1.0,
)
```

## Verify

```python
import logging

def main():
    sentry_sdk.init(...)  # same as above

    logging.debug("I am ignored")
    logging.info("I am a breadcrumb")
    logging.error("I am an event", extra=dict(bar=43))
    logging.exception("An exception happened")

main()
```

- There will be an error event with the message `"I am an event"`.
- `"I am a breadcrumb"` will be attached as a breadcrumb to that event.
- `bar` will end up in the event's `extra` attributes.
- `"An exception happened"` will send the current exception from `sys.exc_info()` with the stack trace and everything to the Sentry Python SDK. If there's no exception, the current stack will be attached.
- The debug message `"I am ignored"` will not surface anywhere. To capture it, you need to lower `level` to `DEBUG` (See below).

## Options

> If you want to change the default behavior of the logging integration, you need to instantiate the integration by hand and pass it to Sentry's `init` function:

Reviewer: ```suggestion
To change the default behavior of the logging integration, instantiate the integration manually and pass it to Sentry's `init` function:
```

</Alert>

## Ignoring a logger

> Sometimes a logger is extremely noisy and spams you with pointless errors. You can completely ignore that logger by calling `ignore_logger`:

Reviewer: ```suggestion
Sometimes a logger is extremely noisy and spams you with pointless errors. You can ignore that logger by calling `ignore_logger`:
```
